src/main/java/nl/knaw/dans/migration/cli/LoadFromDataverseCommand.java

```diff
@@ -115,9 +115,15 @@ else if (uuids != null) {
         }
         else if (file == null) {
             log.info("No DOI(s)/UUIDs provided, loading all datasets");
-            datasetIterator(client, "*").forEachRemaining(
-                item -> proxy.loadFromDataset(((DatasetResultItem) item).getGlobalId())
-            );
+            Iterator<ResultItem> iterator = datasetIterator(client, "*");
```

Comment:
Perhaps add some filer on last version instead.

Reply:
No clue found to filter in https://guides.dataverse.org/en/latest/api/search.html#.